src/main/java/com/ot/restaurant/builders/ProductBuilder.java

```diff
@@ -0,0 +1,35 @@
+package com.ot.restaurant.builders;
+
+import com.ot.restaurant.entities.Ingredients;
+import com.ot.restaurant.entities.Product;
+import java.util.List;
+
+public class ProductBuilder {
```

Comment:
For all builders could make the instance a static method? for example:
```
public class ProductBuilder {
.
.
.
  public static ProductBuilder new(){
     return new ProductBuilder();
  }
}
```

that way we could do:
```
ProductBuilder.new().withName(...)
```

Please follow the same pattern for all the builders